cinema/views.py

```diff
@@ -31,9 +37,30 @@ class CinemaHallViewSet(viewsets.ModelViewSet):
 
 
 class MovieViewSet(viewsets.ModelViewSet):
-    queryset = Movie.objects.all()
+    queryset = Movie.objects.prefetch_related("genres", "actors")
     serializer_class = MovieSerializer
 
+    @staticmethod
+    def _params_to_ints(query_string: str) -> List[int]:
+        return [int(str_id) for str_id in query_string.split(",")]
+
+    def get_queryset(self):
+        queryset = self.queryset
+        title = self.request.GET.get("title")
+        genres = self.request.GET.get("genres")
+        actors = self.request.GET.get("actors")
+
+        if actors:
+            actors = self._params_to_ints(actors)
+            queryset = queryset.filter(actors__id__in=actors)
+        if genres:
+            genres = self._params_to_ints(genres)
+            queryset = queryset.filter(genres__id__in=genres)
```

Comment:
you can use actors__in=actors_ids because for many to many it compares by id by default
Here is example in django docs: https://docs.djangoproject.com/en/5.0/topics/db/examples/many_to_many/
<img width="832" alt="image" src="https://github.com/mate-academy/py-tickets-orders/assets/61380547/c55b0a36-c985-4c30-917e-0ca551d918e3">